packages/react/src/themes/teams/components/List/listStyles.ts

```diff
@@ -1,13 +1,14 @@
 import { debugRoot } from '../../../../styles/debugStyles'
 import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles'
 import { ListProps } from '../../../../components/List/List'
-import { WithAsProp } from '../../../../types'
 
-const listStyles: ComponentSlotStylesPrepared<WithAsProp<ListProps>> = {
+export type ListStylesProps = Pick<ListProps, 'debug' | 'horizontal'> & { isListTag: boolean }
+
+const listStyles: ComponentSlotStylesPrepared<ListStylesProps> = {
   root: ({ props: p }): ICSSInJSStyle => ({
     ...(p.debug && debugRoot()),
     display: p.horizontal ? 'inline-flex' : 'block',
-    ...((p.as === 'ul' || p.as === 'ol') && {
+    ...(p.isListTag && {
```

Comment:
Actually these and the changes inside the listItemStyles are breaking changes...